src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
 // write your code here
+function success() {
+  const message = document.createElement('div');
+
+  message.textContent = 'Promise was resolved!';
+  document.querySelector('body').appendChild(message);
+}
+
+function error() {
+  const message = document.createElement('div');
+
+  message.textContent = 'Promise was rejected!';
+  document.querySelector('body').appendChild(message);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('h1');
```

Comment:
The selector is targeting an `h1` element, but the task requires it to resolve on a click on an element with the class `.logo`. Please change the selector to `document.querySelector('.logo')`.